### ISSUE TITLE
Antsibull-docs 1.2.0 builds all docs instead of just our collection

### ISSUE THREAD

**Issue by LvffY** [ansible-community/antsibull-docs#16]
I'm building my collection doc with *antsibull-docs*.

Since today (03/08/2022) and the release of *antsibull-docs* `1.2.0`, when building my collection docs all ansible collections docs are built (and so fails because I don't have all collections locally.)

## How to reproduce

```bash
mkdir -p ~/tmp/docs
export ANSIBLE_COLLECTIONS_PATH=/path/to/your/collection
antsibull-docs sphinx-init --use-current --dest-dir ~/tmp/docs namespace.collection
cd ~/tmp/docs
pip install --upgrade -r requirements.txt
./build.sh
```

Here you should trying to build documentation for multiple collections and not only yours.

## Workaround

As a temporary workaround you could : 

- Create a constraints file like

```text
antsibull-docs<1.2.0
```

- Then re-run the following commands : 

```bash
mkdir -p ~/tmp/docs
export ANSIBLE_COLLECTIONS_PATH=/path/to/your/collection
antsibull-docs sphinx-init --use-current --dest-dir ~/tmp/docs namespace.collection
cd ~/tmp/docs
pip install --upgrade -r requirements.txt -c /path/to/your/constraints.txt ## Just use your constraint file here, to avoid installing antsibull-docs 1.2.0
./build.sh
```

**Comment by felixfontein** [comment]
This was introduced in https://github.com/ansible-community/antsibull-docs/commit/64a17fda7e9351dde89d4f9c3d73c9415b6d2158#diff-f84a9e8a81549185f5ce78715bbecd163e4faca0bc544471b03888de680d2b15R229, the `collection_names` param must be passed in that line as well.

**Comment by felixfontein** [comment]
Fixed by #17.
